mobile/test/common/integration/base_client_integration_test.h

```diff
@@ -61,17 +63,24 @@ class BaseClientIntegrationTest : public BaseIntegrationTest {
   envoyToMobileHeaders(const Http::TestRequestHeaderMapImpl& request_headers);
   Event::ProvisionalDispatcherPtr dispatcher_ = std::make_unique<Event::ProvisionalDispatcher>();
   envoy_http_callbacks bridge_callbacks_;
-  ConditionalInitializer terminal_callback_;
-  callbacks_called cc_{0, 0, 0, 0, 0, 0, 0, "", &terminal_callback_, {}};
+  ConditionalInitializer* terminal_callback_;
+  std::vector<std::unique_ptr<ConditionalInitializer>> multi_terminal_callbacks_;
+  callbacks_called* cc_;
+  std::vector<std::unique_ptr<callbacks_called>> multi_cc_;
```

Comment:
Maybe `callbacks_called_` and `per_engine_callbacks_called_`? The acronym is a bit ambiguous

Reply:
Maybe I'll expand cc in a later diff? Tons of usages which would clutter this one.